scalafix-testkit/src/main/scala/scalafix/testkit/DiffAssertions.scala

```diff
@@ -70,13 +75,15 @@ trait DiffAssertions extends FunSuiteLike {
   def assertNoDiff(
       obtained: String,
       expected: String,
-      title: String = ""): Boolean = {
+      title: String = ""
+  ): Boolean = {
     val result = compareContents(obtained, expected)
     if (result.isEmpty) true
     else if (obtained.lines.toList == expected.lines.toList) {
       // Ignore \r\n and \n differences
       true
     } else {
+      println(obtained)
```

Comment:
remove ?

Reply:
Good catch, done. I accidentally force pushed so it's not clear from diff that was the only change, sry 😓